### PR TITLE
Awsemo

### PR DESCRIPTION
**Pull request by bielatm**

### CODE REVIEW

File: awsemo/items.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
```
Comment (piotrleniec) [inline awsemo/items.py]: We don't use `AwsemoItem` so we can safely remove this file.

**Comment by piotrleniec** [comment]
`*.pyc` pattern has been added to `.gitignore` which is nice but we still have to remove the remaining `pyc` files.

File: awsemo/middlewares.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
```
Comment (piotrleniec) [inline awsemo/middlewares.py]: I'm not sure about this one but we can experiment and try to remove it. It doesn't provide any functionality we need.

File: awsemo/pipelines.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
```
Comment (piotrleniec) [inline awsemo/pipelines.py]: The same story as in the comment regarding `awsemo/items.py`.

File: awsemo/spiders/awsemo_spider.py [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import scrapy
+
+
+class AwsemoSpider(scrapy.Spider):
+    name = "awsemo"
+    start_urls = ['https://github.com/sindresorhus/awesome']
+
+    def parse(self, response):
+        for link in response.css('div.readme a::attr(href)').extract():
```
Comment (piotrleniec) [inline awsemo/spiders/awsemo_spider.py]: `link` name is a bit misleading because we don't iterate over links (`a` tags) but over URLs.

**Comment by bielatm** [comment]
@piotrleniec fixed

**Review by piotrleniec — APPROVED** [review]

**Pull request merged**